### PR TITLE
Progress bar

### PR DESCRIPTION
**Pull request by yairans-starkware**
### Description of the Changes

* move the progress bar into the multi-choice item
* set min-height to the modal to display the circle loading properly (w/o scrollbar) 
* fix the source selector in small screen

---

### Checklist

- [ ] Changes have been done against `dev` branch, and PR does not conflict.
- [x] Manually tests of the main Application flows are done and passed.
- [x] New unit / functional tests have been added (whenever applicable).
- [x] Automation tests are passing in local environment.
- [ ] Docs have been updated (whenever relevant).
- [ ] PR title is follow the [Conventional Commits](https://www.conventionalcommits.org/) convention: `<type>[optional scope]: <description>`, e.g: `fix: prevent racing of requests`

- - -
This change is [Reviewable](https://reviewable.io/reviews/starkware-libs/starkgate-frontend/288)

### CODE REVIEW

File: src/components/UI/SourceSelect/SourceSelect.theme.js [hunk 1]
```diff
@@ -59,14 +59,15 @@ export const SourceSelectTheme = ({children}) => {
       MuiList: {
         styleOverrides: {
           root: {
-            width: '240px',
-            padding: '8px 16px'
+            width: '240px !important',
```
Comment (galuzan-starkware) [inline src/components/UI/SourceSelect/SourceSelect.theme.js:62]: no need for `!important`

File: src/components/UI/SourceSelect/SourceSelect.theme.js [hunk 2]
```diff
@@ -59,14 +59,15 @@ export const SourceSelectTheme = ({children}) => {
...
+            padding: '8px 16px !important'
```
Comment (galuzan-starkware) [inline src/components/UI/SourceSelect/SourceSelect.theme.js:63]: here too, and I saw more `!important`s so maybe in more places it's redundant

**Review by dan-ziv — COMMENTED** [review]
@yairans-starkware LGTM.
Please check the relevant checkboxes and refer to @galuzan-starkware's comments.

File: src/components/UI/SourceSelect/SourceSelect.theme.js (hunk 2 above)
Comment (yairans-starkware) [reply src/components/UI/SourceSelect/SourceSelect.theme.js:63]: these are needed for small screen where MUI changes the width and padding in the JS

**Review by dan-ziv — APPROVED** [review]

**Pull request merged**